rpc/server.go

```diff
@@ -82,12 +90,26 @@ func NewServer(addr net.Addr, context *Context) *Server {
 	return s
 }
 
-// Register a new method handler. `name` is a qualified name of the form "Service.Name".
-// `handler` is a function that takes an argument of the same type as `reqPrototype`.
-// Both the argument and return value of 'handler' should be a pointer to a protocol
-// message type.
+// Register a new method handler. `name` is a qualified name of the
+// form "Service.Name".  `handler` is a function that takes an
```

Comment:
Some of these aren't even my fault. Emacs users: add `(setq sentence-end-double-space nil)` or emacs may add extra spaces when you format comments with `M-q`. Now I just need to figure out how to write a save hook that runs `canonically-space-region` on every comment.